src/frontend/get-github-setup.ts

```diff
@@ -14,17 +14,39 @@ export default async (req: Request, res: Response): Promise<void> => {
 	req.log.info("Received get github setup page request");
 	const { jiraHost } = res.locals;
 
+	const installationId = req.originalUrl.split("=")[1].split("&")[0]
```

Comment:
We shouldn't parse path parameters from URL manually, because express has functionality for it.
Moreover we shouldn't rely that the installationId will be passed first.

Instead we should do:
`req.query.installationId` (where installationId is the name of the query parameter)

Reply:
- [ ] use query parameters

Reply:
Also, we need to make sure that it won't be possible to pass a wrong InstallationID and then connect other's org. (I guess it won't be because user needs an access to that org, right?)

Reply:
Good call out. Can't connect but it currently fails silently. Will need to chat with Jade to see how we want to display this in the UI.